test/app-tests/loading.js

```diff
@@ -343,6 +346,40 @@ describe('loading:', function () {
             }).done(done, done);
         });
 
+        it("registers correctly with a Riot Team Server", function() {
```

Comment:
This should really be "logs in to the Riot Team Server". I would be awesome to test the slightly more important flow of a user registering with a team email address and being tracked as a referral!